ports/servoshell/Cargo.toml

```diff
@@ -57,7 +57,7 @@ glow = "0.12.2"
 keyboard-types = { workspace = true }
 lazy_static = { workspace = true }
 libc = { workspace = true }
-libservo = { path = "../../components/servo" }
+libservo = { path = "../../components/servo", features=["media-gstreamer"] }
```

Comment:
This is because of https://github.com/servo/servo/issues/30189

Reply:
Hrm...I think this will need to be addressed before this can land. I'm pretty sure we don't want to unconditionally enable this feature.

Reply:
Removed this change from this PR, as issue was resolved in another PR